frontend/server/src/Validators.php

```diff
@@ -325,6 +325,7 @@ public static function validateOptionalDate(
      * @param int|float|null $lowerBound
      * @param int|float|null $upperBound
      * @param boolean   $required
+     * @psalm-assert int $parameter
```

Comment:
```suggestion
     * @psalm-assert numeric $parameter
```

esto _puede_ ser `int`, pero habría que cambiar L341 a [`is_int()`](https://www.php.net/is_int). lo malo es que si esta función recibe cosas de `$r`, no va a ser posible.

Reply:
Listo